src/core/utils/eventutils.js

```diff
@@ -0,0 +1,40 @@
+import FilterType from '../filters/filtertype';
+import AnalyticsEvent from '../analytics/analyticsevent';
+import { AnswersAnalyticsError } from '../errors/errors';
+
+/**
+ * Create the 'REMOVED_FILTER' analytics event, using the given {@link FilterNode}.
+ * @param {Filter} removedFilter The {@link Filter} that was removed.
+ * @param {string} optionType The optionType to send in the request, either STATIC_FILTER, FACET_FILTER, or RADIUS_FILTER.
+ * @param {string} removedFromComponent The component type that removed/reset this filter.
+ * @param {string} verticalKey The current verticalKey
+ * @returns {AnalyticsEvent}
+ */
+export function createRemovedFilterEvent (removedFilter, optionType, removedFromComponent, verticalKey) {
+  const analyticsEvent = new AnalyticsEvent('REMOVED_FILTER');
+  analyticsEvent.addOptions({
+    verticalKey: verticalKey,
+    removedFromComponent: removedFromComponent,
+    optionType: optionType,
+    removedFilter: removedFilter
+  });
+  return analyticsEvent;
+}
+
+/**
+ * Parse the 'optionType' analytics event param.
```

Comment:
We're parsing the filterType, aren't we?

Reply:
yeah somehow what I wrote made sense at the time